schedule/yast/detect_yast2_failures_full.yaml

```diff
@@ -0,0 +1,55 @@
+---
+name: detect_yast2_failures
+description: >
+   Specific test displaying as much dialogs as possible to increase test coverage,
+   which would fail hard in case any yast2 failure is found in logs.
+vars:
+  ASSERT_Y2LOGS:  '1'
```

Comment:
no need for quotes on these values.

Reply:
I have just renamed this file, so I prefer not to edit it. Plus, it's not clear to me weather there is some common agreement in the team about this type of syntax details. We could discuss this in a retro.

Reply:
is not because any agreement, is more following the logic that if it works without those characters, less typing, less possible errors typing, at the end, for Perl would be the same when comparing as boolean. In other cases where we need to escape characters or we use some already taken keyword from yaml we need the quotes AFIK.
But I'm not expert, you can take a look here: http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html. I guess this discussion is similar to method with () or without (). Still you can leave it as it is.